src/package/init.ts

```diff
@@ -16,8 +16,20 @@ type VNodeQueue = VNode[]
 
 const emptyNode = vnode('', {}, [], undefined, undefined)
 
+function sameVnodeSel (vnode1: VNode, vnode2: VNode): boolean {
+  return vnode1.sel === vnode2.sel
+}
+
+function sameVnodeKey (vnode1: VNode, vnode2: VNode): boolean {
+  return vnode1.key === vnode2.key
+}
+
+function sameVnodeIs (vnode1: VNode, vnode2: VNode): boolean {
+  return vnode1.data?.is === vnode2.data?.is
+}
```

Comment:
Since these three functions are all only used a single time I think it would be better to not have them.